numpy/lib/tests/test_function_base.py

```diff
@@ -740,7 +740,7 @@ def test_subclass(self):
                      mask=[[False, False], [True, False],
                            [False, True], [True, True], [False, False]])
         out = diff(x)
-        assert_array_equal(out.data, [[1], [1], [1], [1], [1]])
+        assert_array_equal(out.data, [[1], [4], [6], [8], [1]])
```

Comment:
It is ambiguous what should happen here. All of these values are under the mask.